scripts/test-journal-graph.js

```
@@ -0,0 +1,164 @@
+
+    console.log('--- Dialogue ---');
+    console.log(result.dialogue);
+    console.log('');
+
+    console.log('--- Technical Decisions ---');
+    console.log(result.technicalDecisions);
+    console.log('');
+
+    if (result.errors && result.errors.length > 0) {
+      console.log('--- Errors ---');
+      result.errors.forEach((err) => console.log(`- ${err}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)